src/platform/silabs/NetworkCommissioningWiFiDriver.cpp

```diff
@@ -179,7 +179,8 @@ void SlWiFiDriver::UpdateNetworkingStatus()
     ByteSpan networkId = ByteSpan((const unsigned char *) mStagingNetwork.ssid, mStagingNetwork.ssidLen);
     if (!wfx_is_sta_connected())
     {
-        mpStatusChangeCallback->OnNetworkingStatusChange(Status::kUnknownError, MakeOptional(networkId), NullOptional);
+        mpStatusChangeCallback->OnNetworkingStatusChange(Status::kUnknownError, MakeOptional(networkId),
```

Comment:
@LeelakumarPS The comment for the github issue should be place in the code.

Reply:
Added a comment  mentioning github issue